core/services/chainlink/application.go

```diff
@@ -575,6 +592,10 @@ func (app *ChainlinkApplication) StopIfStarted() error {
 	return nil
 }
 
+func (app *ChainlinkApplication) GetLoopRegistry() *plugins.LoopRegistry {
+	return app.loopRegistry
+}
```

Comment:
Is it necessary/important to have this as a field with a getter?
That is an older pattern - we usually prefer to add things to `app.srvcs` and inject it directly where it is needed, when we can, since it reduces boilerplate.

Reply:
no, probably not, it's only used in the HTTP handler wrapper.

what is the pattern for accessing a subservice from app.srvcs?

inside that HTTP wrapper we need to do the equivalent of 

app.GetLoopRegistry().List/Get

Reply:
IIRC things are directly injected rather than looked up later. It may not fit this case :shrug:

Reply:
looked at the web/router code and it seems that other use cases get around this by accessing app.orm. since the database is not in play for the loop registry, i think i make sense to have a Get'er